### PR TITLE
Remove all bourbon deprecations

### PR DESCRIPTION
**Pull request by maximebiloe**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Switch to webpack 2 and remove all bourbon deprecations
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  |

### CODE REVIEW

File: admin-dev/themes/new-theme/webpack.config.js [hunk 1]
```diff
@@ -1,110 +1,129 @@
...
-    'PrestaKit/dist/js/select2.min.js',
-    'PrestaKit/dist/js/bootstrap-switch.min.js',
-    'PrestaKit/dist/js/jquery.pstagger.min.js',
-    'PrestaKit/dist/js/prestashop-ui-kit.js',
-    'PrestaKit/dist/js/jquery.growl.js',
-    'bootstrap-slider/dist/bootstrap-slider.js',
-    'sprintf-js/src/sprintf.js',
-    './js/theme.js'
-  ],
+let config = {
```
Comment (Quetzacoalt91) [inline admin-dev/themes/new-theme/webpack.config.js]: Error thrown on this line:

```
> new-theme@1.0.0 postinstall /home/thomas/Documents/github/PrestaShop/admin-dev/themes/new-theme
> webpack --progress

/home/thomas/Documents/github/PrestaShop/admin-dev/themes/new-theme/webpack.config.js:5
let config = {
^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at requireConfig (/home/thomas/Documents/github/PrestaShop/admin-dev/themes/new-theme/node_modules/webpack/bin/convert-argv.js:96:18)
    at /home/thomas/Documents/github/PrestaShop/admin-dev/themes/new-theme/node_modules/webpack/bin/convert-argv.js:109:17
    at Array.forEach (native)
```

Comment (Quetzacoalt91) [inline admin-dev/themes/new-theme/webpack.config.js]: Adding 

```js
"use strict";
```

at the beginning of the file fixes the issue.

Comment (maximebiloe) [inline admin-dev/themes/new-theme/webpack.config.js]: It seems that you're using an old version of Node. It should be automatically detected, so as seen with @Shudrum no need to precise it here.

Comment (Quetzacoalt91) [inline admin-dev/themes/new-theme/webpack.config.js]: Yolo then

**Pull request merged**